src/App.js

```diff
@@ -25,31 +27,47 @@ const theme = createTheme({
   },
 });
 
-const categories = ['Books', 'Furniture', 'Electronics', 'Clothing', 'Other'];
-const conditions = ['New', 'Like New', 'Very Good', 'Good', 'Acceptable'];
+const categories = ['Men', 'Women', 'Electronics', 'Clothing', 'Other'];
+const conditions = ['New', 'Like New', 'Very Good', 'Good', 'Acceptable', 'Old', 'Normal'];
```

Comment:
This stuff is gonna be a little bit of a headache to merge—the updated App.js with all of Ashley's refactoring uses a util file to handle all of this stuff. We should be careful around this area when merging

Reply:
Well, I am not sure about this. Can someone help me with this merge request? It seems like the categories and conditions are not in the ListingLayout.js?

Reply:
It might make the most sense for you and Ashley to work through this merge given she made most of the changes. If you look at the main branch though, the src/util.js file has:

```
import { createTheme } from '@mui/material/styles';

export const CATEGORIES = ['Books', 'Furniture', 'Electronics', 'Clothing', 'Other'];
export const CONDITIONS = ['New', 'Like New', 'Very Good', 'Good', 'Acceptable'];
export const SORTING = ['Best Match', 'Price: Low to High', 'Price: High to Low', 'Post Date: New to Old', 'Post Date: Old to New'];
export const MAXPRICE = 500;

export const pageTheme = createTheme({
    palette: {
      primary: {
        main: '#b2102f',
      },
      secondary: {
        main: '#6f252c',
      },
    },
    components: {
      MuiPaper: {
        styleOverrides: {
          root: {
            padding: '15px',
            gap: '5px',
            boxShadow: '0px 3px 5px rgba(0, 0, 0, 0.2)',
          },
        },
      },
    },
  });
```

so we just import with

```
import { CATEGORIES, CONDITIONS, SORTING, MAXPRICE } from './util';
```

in HomePage.js (the new App.js page)